x-pack/plugins/observability_solution/metrics_data_access/common/index.ts

```diff
@@ -28,11 +28,11 @@ export {
   InventoryVisTypeRT,
   ItemTypeRT,
   SnapshotMetricTypeRT,
-  ESSumBucketAggRT,
-  ESDerivativeAggRT,
-  ESBasicMetricAggRT,
   SnapshotMetricTypeKeys,
   ESTermsWithAggregationRT,
+  isDerivativeAgg,
+  isBasicMetricAgg,
+  isSumBucketAgg,
 } from './inventory_models/types';
```

Comment:
Should these be in the `types` file? I wonder if it would be better if we created a `utils` file or something like that. wdyt?